content_sync/pipelines/definitions/concourse/site-pipeline.yml

```diff
@@ -219,8 +246,8 @@ jobs:
             args:
             - -exc
             - |
-              aws s3 sync s3://((ocw-studio-bucket))/((s3-path)) s3://((ocw-bucket))/((site-url)) --metadata site-id=((site-name))              
-              aws s3 sync course-markdown/public s3://((ocw-bucket))/((base-url)) --metadata site-id=((site-name))
+              aws s3((cli-endpoint-url)) sync s3://((ocw-studio-bucket))/((s3-path)) s3://((ocw-bucket))/((site-url)) --metadata site-id=((site-name))              
+              aws s3((cli-endpoint-url)) sync course-markdown/public s3://((ocw-bucket))/((base-url)) --metadata site-id=((site-name))
         on_failure:
```

Comment:
One minor issue I noticed is that there is no webhook sent back on dev when the pipeline completes successfully because that is only included in the `clear-cdn-cache` step.  Could get around that by adding another `DEV-ONLY` section with that webhook call added here just before `on_failure.  

```
        on_success:
          try:
            do:
              - put: ocw-studio-webhook
                timeout: 1m
                attempts: 3
                params:
                  text: |
                    {
                      "version": "((pipeline_name))",
                      "status": "succeeded"
                    }
```                    

<img width="170" alt="Screen Shot 2022-07-27 at 8 45 48 AM" src="https://user-images.githubusercontent.com/187676/181250030-4a8f5ad4-94ad-46aa-9558-cf2e5d1cfed3.png">